AUTHORS

```diff
@@ -0,0 +1,13 @@
+Backscrub (previously Deepbacksub) was conceived of and implemented by
+Florian Echtler <floe@butterbrot.org> in early 2020.
+
+In surname alphabetical order, the following people have also contributed to
+deepbacksub:
+
+  Phil Ashby <phil.github@ashbysoft.com>
+  Ismael Barros² <razielmine@gmail.com>
```

Comment:
Is the `²` a footnote? Where's it specified?

Reply:
Not a footnote, it's their current Github name.